src/ChannelAdam.Soap/Soap11Maker.cs

```diff
@@ -80,23 +80,23 @@ public static XElement CreateSoapBodyWithFault(Soap11FaultCode code, string faul
         #region CreateSoapFault
 
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1720:IdentifiersShouldNotContainTypeNames", MessageId = "string", Justification = "As per the SOAP specification.")]
-        public static XElement CreateSoapFault(Soap11FaultCode code, string faultString)
+        public static XElement CreateSoapFault(Soap11FaultCode code, string faultString, string? prefix = null)
```

Comment:
Don't we need to do this above - in CreateSoapEnvelopeWithFault() and CreateSoapBodyWithFault() - passing the prefix into both CreateSoapBody() and CreateSoapFault()?

Reply:
Yep, seems like I forgot to add the passing parameter into CreateSoapFault.